lambda_function.py

```diff
@@ -65,5 +65,7 @@ def lambda_handler(event, context):
 
     # Valid event, and one we are interested in
     cluster = event["detail"]["clusterArn"]
-    adjust_service_desired_count(ecs_client(), cluster, service)
+    serviceItems = service.split(',')
```

Comment:
Please remain consistent with the python naming convention of `lower_case_with_underscores` for variables.